.changeset/tasty-taxis-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/abi-coder": patch
+---
+
+encoding bugfix
```

Comment:
This PR must exclude work from other PRs.

If you need things from https://github.com/FuelLabs/fuels-ts/pull/1107, you must make `1106-bug-nested-implicit-generics-fail-encoding` the base branch for https://github.com/FuelLabs/fuels-ts/pull/1116 and update this PR accordingly to dedupe things.

Reply:
This is a leftover from that PR that I didn't remove.